Makefile

```diff
@@ -30,6 +30,10 @@ lint:
 build: create-build-dir lint unit-test
 	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/bm-inventory cmd/main.go
 
+.PHONY: build-minimal
+build-minimal: create-build-dir
+	CGO_ENABLED=0 go build -o $(BUILD_FOLDER)/bm-inventory cmd/main.go
```

Comment:
We can probably refactor it later so 'build' will use 'build-minimal', that way we can avoid duplicate code (lines 31 and 35).